.gitignore

```diff
@@ -1,11 +1,16 @@
 .DS_Store
 
 # dev files
-/target/
+build
+target/
 platform/*.a
+platform/*.lib
 platform/*.tar.br
 
 # examples
-/examples/main
-/examples/output/
-/examples/**/*.html
\ No newline at end of file
+example/main
+example/output/
+example/**/*.html
+
+# generated files
+crates/roc_std/
```

Comment:
Seems like `roc_std` is still in this PR, you may need to `git rm` the directory.